src/_P004_Dallas.ino

```diff
@@ -77,6 +77,8 @@ boolean Plugin_004(byte function, struct EventStruct *event, String& string)
 
     case PLUGIN_WEBFORM_LOAD:
     {
+      addFormNote(F("External pull up resistor is needed, see docs!"));
+
```

Comment:
@TD-er should be more like: 
```
if (Plugin_004_DallasPin != -1) {
  if (digitalRead(Plugin_004_DallasPin) != HIGH) {
    addFormNote(F("External pull up resistor is needed, see docs!"));
  }
 ...
```

Reply:
An internal pull up could be active here, which may not be enough as stated in the issue discussion.